InOut/CMakeLists.txt

```diff
@@ -110,37 +105,21 @@ if(USE_PULSEAUDIO)
     make_plugin(rtpulse rtpulse.c PulseAudio::PulseAudio)
 endif()
 
-if(PORTAUDIO_FOUND OR USE_VCPKG)
-    message(STATUS "Building rtpa module.")
+check_deps(USE_PORTAUDIO portaudio_FOUND)
+if(USE_PORTAUDIO)
     make_plugin(rtpa rtpa.c)
-    if(USE_VCPKG)
+    # use the static library if available
+    if (TARGET portaudio_static)
         target_link_libraries(rtpa PRIVATE portaudio_static)
     else()
-        target_include_directories(rtpa PRIVATE ${PORTAUDIO_INCLUDE_DIRS})
-        target_link_libraries(rtpa PRIVATE ${PORTAUDIO_LIBRARIES})
+        target_link_libraries(rtpa PRIVATE portaudio)
```

Comment:
Can add the targets like this instead to avoid duplication
```
target_link_libraries(rtpa PRIVATE
            $<IF:$<TARGET_EXISTS:portaudio>,portaudio,portaudio_static>
)
```

Reply:
Nice, changed it to that, thanks!